### PR TITLE
Finished the murder mystery ! :) - Luke Brannagan

### PR DESCRIPTION
**Pull request by Nevnet99**
Found out who the murder is. ! :+1: :policeman:.

### CODE REVIEW

**Comment by harrim91** [comment]
good stuff! you seem to have a random file committed too - not sure what that is - but you solved the mystery and were the first to open a PR so you get a trophy 🏆

**Comment by Nevnet99** [comment]
Wahoo I can't remember I think I made membership groups to add the memberships he was apart of into one file and grep how many times his name came up 🤣

**Comment by Nevnet99** [comment]
@harrim91  still got my trophy 😂 🏆

File: mystery/memberships/membershipgroup [hunk 1]
```diff
@@ -0,0 +1,5176 @@
+Courtney Yankey
```
[the remark this answers is not in the document]
Comment (Nevnet99) [reply mystery/memberships/membershipgroup:1]:
```suggestion

```

**Pull request closed without merging**